src/main/webapp/populateData.js

```diff
@@ -104,6 +116,14 @@ function populateTasks() {
         return response.json();
       })
       .then((tasksResponse) => {
+        const taskListNames = tasksResponse['taskListNames'];
```

Comment:
nit: use `allTaskListNames` for consistency

Reply:
Good idea - thanks!

Reply:
`RESOLVED`